src/test/resources/application-test.properties

```diff
@@ -18,7 +20,7 @@ daps.keystore=resources/iais3-keystore.jks
 #daps.keystore=resources/isstbroker-keystore.jks
 daps.keystorePwd=password
 daps.keystoreAlias=1
-daps.verify=true
+daps.verify=false
```

Comment:
Did you intentionally turn off the DAPS validation? I guess this should be re-enabled

Reply:
This wont work for the automated online tests right now. Therefore I would propose to leave this currently as false.